src/stories/Other/Breakdown.stories.js

```diff
@@ -54,6 +54,10 @@ export const Default = () => {
                     This is a really long message that should wrap somehow
                 </Breakdown.Item>
 
+                <Breakdown.Item secondary="07/23/2021" info={<EmptyComponent />} value={0} methodIcon={<CardIcon />}>
+                    LongMessageThat ShouldOnlyBreakAt AWhitespaceLoremIpsum
+                </Breakdown.Item>
```

Comment:
Given the string:

```
LongMessageThat ShouldOnlyBreakAt AWhitespaceLoremIpsum
```

### Bug

It breaks incorrectly in the middle of "ShouldOnly"

<img src="https://github.com/xola/ui-kit/assets/65708/71d231e2-6feb-4fcd-b8f1-45066f3dea80" height="500" />

### Solution

The correct way is to break at the whitespace after "ShouldOnlyBreakAt"

<img src="https://github.com/xola/ui-kit/assets/65708/b240ac3a-d557-49ec-9dd1-c3cbbb5d0a03" height="500" />